### PR TITLE
fix: invalid cron schedules

### PR DESCRIPTION
**Pull request by andreabadesso**
### Acceptance Criteria
- Cronjobs were not being executed because of the handler definition
- Now the `cleanUnsentTxProposalsUtxos` will not fail if there are no `txProposals` to release

### Security Checklist
- [X] Make sure you do not include new dependencies in the project unless strictly necessary and do not include dev-dependencies as production ones. More dependencies increase the possibility of one of them being hijacked and affecting us.

### CODE REVIEW

File: tests/utils/pushnotification.utils.test.ts [hunk 1]
```diff
@@ -55,7 +55,7 @@ describe('PushNotificationUtils', () => {
     });
```
Comment (andreabadesso) [inline tests/utils/pushnotification.utils.test.ts:55]: Changes on this file are unrelated to the PR, but were yielding a lot of lint errors on the deploy

**Review by alexruzenhack — APPROVED** [review]

**Comment by alexruzenhack** [comment]
Should we have debug logs like `Starting execution` or `Ending with success` just to have some logs to watch for? Does it worth something as a practice or not?

**Comment by andreabadesso** [comment]
> Should we have debug logs like Starting execution or Ending with success just to have some logs to watch for? Does it worth something as a practice or not?

We already have those, AWS logs when the requests starts and ends:

image

**Comment by alexruzenhack** [comment]
> > Should we have debug logs like Starting execution or Ending with success just to have some logs to watch for? Does it worth something as a practice or not?
> 
> We already have those, AWS logs when the requests starts and ends:
> 
> image

Ok. I will apply this in further situations as well.

File: src/db/cronRoutines.ts [hunk 2]
```diff
@@ -59,10 +59,18 @@ export const cleanUnsentTxProposalsUtxos = async (): Promise<void> => {
   const txProposalsBefore = now - STALE_TX_PROPOSAL_INTERVAL;
   const unsentTxProposals: string[] = await getUnsentTxProposals(mysql, txProposalsBefore);
 
-  try {
-    await releaseTxProposalUtxos(mysql, unsentTxProposals);
-    await updateTxProposal(mysql, unsentTxProposals, now, TxProposalStatus.CANCELLED);
-  } catch (e) {
-    logger.error('Failed to release unspent tx proposals: ', unsentTxProposals);
+  if (unsentTxProposals.length > 0) {
```
Comment (tuliomir) [inline src/db/cronRoutines.ts:62]: I think it's worth mentioning this added validation on the PR's _Acceptance Criteria_

**Review by tuliomir — APPROVED** [review]

File: src/db/cronRoutines.ts (hunk 2 above)
Comment (andreabadesso) [reply src/db/cronRoutines.ts:62]: Done, thanks

**Pull request merged**